### ISSUE TITLE
Should browsers support ContentEditable for <option> elements?

### ISSUE THREAD

**Issue by Leo7k** [w3c/editing#180]
Currently ContentEditable for _option_ elements has no support in most popular browsers. Should it be supported? Is it correct use case for ContentEditable?

**Comment by gked** [comment]
@Leo7k what are you trying to achieve with making option contenteditable? What is the use case here?

**Comment by Leo7k** [comment]
@gked
I am trying to include editable <option>s into <select>

**Comment by Leo7k** [comment]
@gked, I am trying to include editable options into select.. I want to let users to fill in option text.

**Comment by johanneswilm** [comment]
contenteidtable is normally used for richtext editing, so I would think this is a outside of normal use cases and should probably be addressed by a JavaScript library that emulates a select/option combination.

**Comment by gked** [comment]
@johanneswilm indeed, it is not something that we considered before when it comes to CE. Allowing the framework to do this sounds like a reasonable approach.
On the other hand, it does fall into the theme where we are trying to expose hooks to web developers to modify web content more easily. So perhaps, instead of making it CE, we could expose some JS API to allow them to add/remove/edit option elements and I would probably be more efficient than creating new elements and apppending them to the existing conrtol, the way it is currently done on the web. But I don't have a strong preference just considering different points of view.  @Leo7k sounds like you don't have a strong opinion on this either?
